docs_app/src/app/custom-elements/migration-timeline/data-access/migration-timeline-struckture/README.md

```diff
@@ -0,0 +1,182 @@
+# Data structure for the Migration-Timeline
+The structure aims to be autogenerated from the code in the future.
+
```

Comment:
please fix typo `struckture` in folder

Reply:
fixed here: https://github.com/ReactiveX/rxjs/pull/5128/commits/be5e41e4ddbffd2d709e4910b2108b7f848fb0a1